sql/python/sqlflow_submitter/maxcompute.py

```diff
@@ -0,0 +1,45 @@
+from odps import ODPS, tunnel
+
+# Copyright 2019 The SQLFlow Authors. All rights reserved.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
```

Comment:
Copyright section comes after the import. Lets manually adjust this before https://github.com/sql-machine-learning/sqlflow/issues/459 is fixed.